app/src/main/java/org/astraea/app/partitioner/Dispatcher.java

```diff
@@ -47,6 +54,39 @@ public interface Dispatcher extends Partitioner {
    */
   default void configure(Configuration config) {}
 
+  /**
+   * Use the producer to get the scheduler, allowing you to control it for interdependent
+   * messages.Interdependent message will be sent to the same partition. The system will
+   * automatically select the node with the best current condition as the target node. For example:
+   *
+   * <pre>{
+   * @Code
+   * Dispatch.startInterdependent(producer);
+   * producer.send();
+   * Dispatch.endInterdependent(producer);
+   * }</pre>
+   *
+   * Begin interdependence function.Let the next messages be interdependent.
+   *
+   * @param producer Kafka producer
+   */
+  static void beginInterdependent(Producer<Key, Value> producer) {
+    dispatcher(producer).begin();
+  }
+
+  /**
+   * Close interdependence function.Send data using the original Dispatcher logic.
+   *
+   * @param producer Kafka producer
+   */
+  static void endInterdependent(Producer<Key, Value> producer) {
+    dispatcher(producer).end();
+  }
+
+  private static Dispatcher dispatcher(Producer<Key, Value> producer) {
+    return (Dispatcher) Utils.reflectionAttribute(producer, "partitioner");
```

Comment:
It should throw meaningful exception when the partitioner is not Astraea dispatcher.